src/Payloads/Refund.php

```diff
@@ -136,15 +137,25 @@ public function setComment($comment) {
 
     /**
      * @return array
+     * @throws ParametersException
      */
     public function getRequestBody() {
+        if ($this->getAmount() === 0) {
+            throw new ParametersException('Refund warning, the refund is zero');
```

Comment:
maybe we need to have tickets in our cms to catch this new error, or the ones below